hazelcast/src/test/java/com/hazelcast/config/YamlConfigBuilderTest.java

```diff
@@ -2919,39 +2919,6 @@ public void testHotRestartEncryptionAtRest_whenVault() {
         assertEquals("TLS", sslConfig.getProperties().get("protocol"));
     }
 
-    @Override
-    @Test
-    public void testMapEvictionPolicyClassName() {
-        String mapEvictionPolicyClassName = "com.hazelcast.map.eviction.LRUEvictionPolicy";
-        String yaml = ""
-                + "hazelcast:\n"
-                + "  map:\n"
-                + "    test:\n"
-                + "      map-eviction-policy-class-name: " + mapEvictionPolicyClassName;
```

Comment:
same as above for `XmlConfigBuilderTest`

Reply:
added a specific test for this case.